test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -6362,7 +6362,7 @@ def manager(self, cmd, obj_type, attrib=None, id=None, extend=None,
             if rc == 0:
                 if cmd == MGR_CMD_LIST:
                     bsl = self.utils.convert_to_dictlist(ret['out'], attrib,
-                                                         mergelines=False)
+                                                         mergelines=True)
```

Comment:
@anamikau Even I don't know why it was False in first place... I just quick checked first version of PTL in P4 and there also it is False so I believe previously we never got as any output of qmgr list command which has one line wrapped to multiple line...

Reply:
I can't explain why it was false to begin with. It was like that when it was handed off. The event list is split into multiple lines when it gets long. I have not filed a bug because I don't think PTL was doing anything wrong.